clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/config/KubernetesConfigurationProperties.java

```diff
@@ -75,4 +85,93 @@ public void validate() {
       }
     }
   }
+
+  public KubernetesConfigurationProperties() {}
+
+  /**
+   * For larger number of Kubernetes accounts, SpringBoot based properties binding is taking tens of
```

Comment:
I think it's worth to document the limitations of this approach (in case they apply):
* Property placeholders don't work (for example you cannot do `kubeconfigFile: ${KUBECONFIGS_HOME}/kubecfg-dev`)
* `encryptedFile:` and `encrypted:` notation doesn't work
* `configserver:` only works for kubeconfig files

Reply:
Included all the limitations in the BootstrapKubernetesConfigurationProvider class.